.travis.yml

```diff
@@ -59,8 +51,7 @@ matrix:
 
 stages:
     - precheck
-    - main
-    - extras
+      if: type = push
```

Comment:
Having approved this... it looks like Travis isn't building master anymore.

So [maybe `if: repo = HypothesisWorks/hypothesis AND branch = master`](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/) ?  I hope?  "dealing with CI" is not my favorite :sob:

Reply:
I'm pretty sure what I wrote *should* work as it's the same check that guards deploys! Ugh.

Reply:
Hopefully #2142 will fix the issue